oss-fuzz.sh

```diff
@@ -32,11 +32,7 @@ function compile_fuzzer {
   fuzzer=$3
   corpusfile="${path}/testdata/${fuzzer}_seed_corpus.zip"
   echo "Building $fuzzer (expecting corpus at $corpusfile)"
-  (cd $path && \
-        go-fuzz -func $func -o $WORK/$fuzzer.a . && \
-        echo "First stage built OK" && \
-        $CXX $CXXFLAGS $LIB_FUZZING_ENGINE $WORK/$fuzzer.a -o $OUT/$fuzzer && \
-        echo "Second stage built ok" )
+  compile_go_fuzzer $path $func $fuzzer
```

Comment:
I believe there are two changes needed. In this file, using
```suggestion
  compile_go_fuzzer github.com/ethereum/go-ethereum/$1 $2 $3
```
And in the dockerfile, change it to 
```
FROM gcr.io/oss-fuzz-base/base-builder

RUN go get github.com/ethereum/go-ethereum

RUN cp $GOPATH/src/github.com/ethereum/go-ethereum/oss-fuzz.sh $SRC/build.sh

WORKDIR $SRC/
```

Reply:
Indeed @holiman I forgot the change in the Dockerfile to put go-ethereum in `$GOPATH/src`

Reply:
There are a couple of extra steps needed, but I think I'm pretty close to having it figured out now, will have a PR ready tomorrow